.eslintrc.js

```diff
@@ -46,7 +46,9 @@ module.exports = {
     'react/display-name': 'off',
     'react-hooks/exhaustive-deps': 'warn',
     'react/jsx-one-expression-per-line': 'off',
+    'react/jsx-curly-brace-presence': 2,
```

Comment:
prefer the more verbose format so one doesn't need to remember how to map numbers back to them
```suggestion
    'react/jsx-curly-brace-presence': 'error',
```

Reply:
ok